metatron-app/metatron-discovery
# fix error dashboard

### Description
Edit the following
1. Widget edit button operation error
2. Fix chart y-axis panel error
3. List update error during replication
4. List update error when deleting

**Related Issue** : 


### How Has This Been Tested?

1. Check that the chart edit screen is displayed immediately when the chart edit button is pressed.
2. Check if the y-axis panel is displayed normally when opening the chart editing screen.
3. Check if the list is displayed normally when cloning the dashboard.
4. Check if the list count is displayed normally when deleting the dashboard.

#### Need additional checks?


### Types of changes



### Checklist:


### Additional Context
